scripts/importTemplates.js

```diff
@@ -117,55 +105,91 @@ async function main() {
+
+    console.log('Validating template...');
+    const query = validateTemplate(JSON.stringify(json));
+    console.log(query);
+    const validateTemplateResult = await axiosDev
+      .post(grapqlImportUrl, {
+        query: query,
+      })
+      .catch((e) => console.log(e));
 
     const validationResult =
       validateTemplateResult.data.data.validateTemplateImport.validationResult;
+    console.log(
+      `validation result: ${JSON.stringify(validationResult, null, 2)}`
+    );
 
-    const templateConflictResolution = validationResult.questionComparisons.map(
-      (comparison) => {
+    const templateConflictResolution =
+      validationResult.validationData.questionComparisons.map((comparison) => {
         const question = comparison.newQuestion;
 
         return {
           itemId: question.id,
           strategy: 'USE_NEW',
         };
-      }
-    );
-
-    const subTemplatesConflictResolutions =
-      validationResult.subTemplatesWithValidation.map((template) => {
-        return template.questionComparisons.map((comparison) => {
-          const question = comparison.newQuestion;
-
-          return {
-            itemId: question.id,
-            strategy: 'USE_NEW',
-          };
-        });
       });
 
-    const importTemplateResult = await axiosDev.post(grapqlImportUrl, {
-      query: importTemplate,
-      variables: {
-        conflictResolutions: templateConflictResolution,
-        subTemplatesConflictResolutions: subTemplatesConflictResolutions,
-        templateAsJson: json,
-      },
-    });
+    let subTemplatesConflictResolutions;
+    if (
+      validationResult.validationData.subTemplateValidationData !== undefined &&
+      validationResult.validationData.subTemplateValidationData !== null
+    ) {
+      subTemplatesConflictResolutions =
+        validationResult.validationData.subTemplateValidationData.map(
+          (template) => {
+            template.questionComparisons = template.questionComparisons ?? [];
+
+            return template.questionComparisons.map((comparison) => {
+              const question = comparison.newQuestion;
+
+              return {
+                itemId: question.id,
+                strategy: 'USE_NEW',
+              };
+            });
+          }
+        );
+    }
+
+    console.log('Importing template...');
+    const importTemplateResult = await axiosDev
+      .post(grapqlImportUrl, {
+        query: importTemplate,
+        variables: {
+          conflictResolutions: templateConflictResolution,
+          subTemplatesConflictResolutions: subTemplatesConflictResolutions,
+          templateAsJson: json,
+        },
+      })
+      .catch((e) => console.log('Failed to import template, \n' + e));
 
     console.log('Template import result:');
     console.log(importTemplateResult.data.data);
-  });
+    console.log('------------------------------------------------------------');
+  }
 }
 
-main();
+main().catch((e) => console.error(e));
```

Comment:
There's a couple of other promise `.catch` methods that call `console.log` Should they be changed to `error` too?